libraries/core/src/main/res/values/strings.xml

```diff
@@ -121,4 +123,6 @@
     <string name="results_failure">Failed to begin tracking series %s, please try again…</string>
 
     <string name="error_generic">An unknown error has occurred…</string>
+    <string name="anime">Anime</string>
```

Comment:
Instead of adding the `anime` and `manga` strings I think it would make more sense to use the `nav_anime` and `nav_manga` strings, tying the setting to the string